src/components/sbb-teaser-hero/sbb-teaser-hero.stories.js

```diff
@@ -1,71 +1,113 @@
 import { h } from 'jsx-dom';
-import images from '../../global/images';
 import readme from './readme.md';
 
-// --- Controls
-const imageLoading = {
-  control: {
-    type: 'inline-radio',
-  },
-  options: ['eager', 'lazy'],
-  table: {
-    category: 'Performance',
-  },
-};
-
 const openInNewWindowControl = {
   control: {
     type: 'inline-radio',
   },
   options: ['true', 'false'],
 };
 
+/* --- sbb-text slot ----------------------------------- */
+
+const sbbPanelTextArgs = {
+  text: 'Rücksichtsvoll mit SBB Green Class',
+};
+
+const SlotSbbPanelTextTemplate = (args) => <p slot="text">{args.text}</p>;
+
+/* --- sbb-panel - link slot ----------------------------------- */
+
+const sbbPanelLinkArgs = {
+  href: 'https://www.sbb.ch/',
+  text: 'Mehr erfahren',
+};
+
+const SlotPanelSbbLinkTemplate = (args) => (
+  <sbb-link
+    slot="link"
+    href-value={args.href}
+    icon="chevron-small-right-small"
+    icon-flip=""
+    icon-placement="end"
```

Comment:
icon, icon-placement, text-size and variant should also be configured automatically by sbb-panel (icon would be much easier with the not yet ready sbb-icon component)

Reply:
Yes, that should be fine